pkg/cache/fake.go

```diff
@@ -2,11 +2,11 @@ package cache
 
 import (
 	"encoding/json"
-	"k8s.io/api/core/v1"
 	"strconv"
 	"sync"
 
 	"github.com/aws/amazon-eks-pod-identity-webhook/pkg"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Small nitpick but this should probably be moved in the above section. Otherwise it will probably fail `goimports --local` validation